build.gradle

```diff
@@ -51,6 +51,8 @@ dependencies {
 
 	implementation 'ca.uhn.hapi.fhir:hapi-fhir-base:5.2.0'
 	implementation 'ca.uhn.hapi.fhir:hapi-fhir-structures-dstu3:5.2.0'
+	implementation 'org.springframework:spring-jms:5.2.6.RELEASE'
+	implementation 'org.apache.qpid:qpid-jms-client:0.51.0'
```

Comment:
There's a newer version of this dependency we could use, `v.0.56.0`

https://mvnrepository.com/artifact/org.apache.qpid/qpid-jms-client/0.56.0

Reply:
Sure, fixed in b2c7fe13fb31dcc8518c2aecd30d1d8ea41ce007 . Thanks!